engines/sword25/configure.engine

```diff
@@ -1,3 +1,3 @@
 # This file is included from the main "configure" script
 # add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps] [components]
-add_engine sword25 "Broken Sword 2.5" yes "" "" "png 16bit highres theoradec" "lua theoradec"
+add_engine sword25 "Broken Sword 2.5" yes "" "" "png 16bit highres theoradec lua" ""
```

Comment:
theoradec is not a soft dependency but a hard one

Reply:
I checked that code specifically. It is referred from `fmv/movieplayer.h` and the code generates STUBs in case of the missing codec: https://github.com/scummvm/scummvm/blob/master/engines/sword25/fmv/movieplayer.cpp#L177